OnboardPietroPieri/components/loginscreen.tsx

```diff
@@ -45,10 +47,22 @@ function TextField({
   );
 }
 
-function LoginButton({ body, onPress }: LoginButtonProps): React.JSX.Element {
+function LoginButton({
```

Comment:
Acho importante criar um componente separado para botão, mas, olhando o código do componente, não existe nada que esteja particularizando para o caso de login. Esse componente seria mais útil se você trocasse o nome para `Button` e sempre usasse ele quando for fazer um botão no app.

Ao criar esse componente genérico, você garante que todos os botões do app terão a mesma aparência, o mesmo comportamento de loading, ...